net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="A small dynamic dns client with multiple servers support and a high flexibility"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers support and a high flexibility."
+HOMEPAGE="https://yaddns.github.io/"
+COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/yaddns/yaddns/archive/yaddns-$portVersion.tar.gz"
```

Comment:
SourceForge (or any mirror) should be preferred over Github as Github release archives are non-deterministic (generated everytime they're downloaded)

```diff
-SOURCE_URI="https://github.com/yaddns/yaddns/archive/yaddns-$portVersion.tar.gz"
+SOURCE_URI="https://downloads.sourceforge.net/yaddns/yaddns-$portVersion.tar.gz"
```

Reply:
Hmm, I new that GitHub archives were generated dynamically, but I thought they were determinist.
I'm quite sure about that, because we have many recipes that use dynamic archives from github and we are able to add a stable checksum.

Reply:
Tags are movable, but most project generally don't do so, so yes, for this port they're quite deterministic. However, as SourceForge have mirrors all around the world, it would greatly help with download speed :)

Reply:
SourceForge has more redirections than GitHub does, which makes it practically slow in most cases. Also, don't underestimate GitHub's CDN :)

Plus, SourceForge is not 100% deterministic because projects can be deleted, and interestingly that's more often (to me) than tags being moved on GitHub.

Reply:
Oops, I realize I had written a comment yesterday but forgot to submit it, and lost it. So here it is.
I was suggesting to switch SOURCE_URI to SourceForge, mainly for the same reason as @alaviss:
```
SOURCE_URI="https://downloads.sourceforge.net/yaddns/yaddns-$portVersion.tar.gz"
```
Another advantage is that you have a `configure` script already ready, and probably usable without having to call `autogen.sh`.

BYW, I think I yaddns won't build with gcc2, so you need to add support for the x86 secondary arch.

You do this with:
```
ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
SECONDARY_ARCHITECTURES="x86"
```
and then adding $secondaryArchSuffix at various places, like `haiku` -> `haiku$secondaryArchSuffix`.
Reminder: `haiku_devel` should be changed to `haiku${secondaryArchSuffix}_devel`.

Reply:
> Another advantage is that you have a configure script already ready, and probably usable without having to call autogen.sh.

That's a good point. I'll do it.

Reply:
@fbrosson ... and `commandBinDir` too?

Reply:
Late answer: Yes, the commandBinDir is OK, so thanks for having added it :)

**Note** for the visitors reading this thread: we had to drop the SourceForge tarball and switch back to the GitHub archive, so most of the discussion above is outdated.